src/neovim/input.ts

```diff
@@ -390,4 +433,12 @@ export default class NeovimInput {
         this.element.style.left = x + 'px';
         this.element.style.top = y + 'px';
     }
+
+    updateFontSize() {
+        const {specified_px} = this.store.font_attr;
+
+        const font_size = specified_px * (window.devicePixelRatio || 1);
```

Comment:
I think that this is a problem related to device pixel ratio. Changing this line to `const font_size = specified_px;` may fix this (don't need to consider pixel ratio for DOM element).

Reply:
Yeah, as you said, it seems a problem of device pixel ratio because it's twice bigger than line height on my environment (device pixel ratio is 2).